pkg/controllers/dynakube/controller_system_test.go

```diff
@@ -0,0 +1,815 @@
+package dynakube
```

Comment:
I hoped that the contents of this file will be dissected and moved to packages that they actually test and be replaced by simpler test for this package that test the relevant parts for the `dynakube-controller`

now instead of removing/replacing the fragile tests, we just moved them a bit

Reply:
@0sewa0 I agree with you, so should we move them to sub-reconcilers?
It's unclear what we want here before we merge PR?

Reply:
yeah, i'll create a story about this file.